### DISCUSSION TITLE
project doesn't run when I use --enable-bcc

### DISCUSSION THREAD

**Discussion by ddi98** [dashingsoft/pyarmor#1145] [Q&A]
hello, as per the title when i obfuscate my project using --enable-bcc the obfuscation process gets done but the script doesn't run (we're talking about the same project i sent you via email)

**Comment by jondy** [comment]
Try `8.1.dev9`

**Comment by ddi98** [comment]
generated this warning during obfuscation:
```
.pyarmor\bcc\FormulariTools.utils.c:1525:14: warning: incompatible pointer types assigning to 'PyObject *' (aka 'struct _object *') from 'PyObject *[1]' [-Wincompatible-pointer-types]
  ctx_consts = ((PyTupleObject *)ctx_consts)->ob_item;
             ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning generated.
```

if I try to start the app the flask web server also starts but then the process ends by itself

**Comment by jondy** [comment]
I have found a bug related to this, I'll check it fixed in next release.

**Comment by jondy** [comment]
Please try pyarmor `8.1.1a1` with pyarmor.cli.core `2.1.a1`

I have fixed a bcc bug

**Comment by ddi98** [comment]
now the flask server starts but when i try to login in my app it terminates the process

**Reply by jondy** [reply]
I have fixed another bcc bug in `8.1.5`, you can try this version.

**Reply by ddi98** [reply]
Hi, I still have the same problem today
when I use enable-bcc the web server starts and then crashes itself after a few seconds
